lib/api/controllers/documentController.js

```diff
@@ -716,7 +716,7 @@ class DocumentController extends NativeController {
    * @returns {Promise.<Object>} { successes, errors }
    */
   async _mChanges (request, methodName, action) {
-    let _source = true;
+    let _source = request.input.args._source !== undefined ? request.getBoolean('_source') : true;
```

Comment:
Why are you not just using `getBoolean` like `document.update` for example?

Reply:
Because if the value is `undefined`, the `getBoolean()` returns `false` instead of `true` the value I want from the issue #2073